### ISSUE TITLE
zsh autocompletion

### ISSUE THREAD

**Issue by CRTified** [Amarandus/zsh-johnnydecimal#1]
It would be nice to have auto completion for categories and unique identifiers.

Functions to add autocompletion:

- [x] `jcd`
- [x] `jcp`
- [x] `jmv`
- [ ] `jmkuni`

**Comment by CRTified** [comment]
First step is done with https://github.com/Amarandus/zsh-johnnydecimal/commit/dc6043b496a8645187387fa58d8e7f5998481079

It would be desirable to separate the completion of the Category from the completion of the unique ID.

**Comment by rlam3** [comment]
Is this project still active?

**Comment by CRTified** [comment]
Yes and no - it's "good enough" for my usage, so its priority dropped a bit. If you have any urgent or important issues or feature requests, feel free to create a new issue.
